scripts/Get-Tags.ps1

```diff
@@ -0,0 +1,17 @@
+param(
+    [string]$Branch="master",
+    [string]$ImageBuilderImageName="microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171005132855"
+)
+
+$ErrorActionPreference = 'Stop'
+$repoRoot = Split-Path -Path "$PSScriptRoot" -Parent
+
+& docker pull $ImageBuilderImageName
+
+& docker run --rm `
+    -v /var/run/docker.sock:/var/run/docker.sock `
+    -v "${repoRoot}:/repo" `
+    -w /repo `
+    $ImageBuilderImageName `
+    generateTagsReadme `
```

Comment:
I personally would prefer the last two lines combines as they are not individual docker run args.  The grouping is logically a single arg which is the cmd being executed within the container.  I find it easier to read as one.

Reply:
Agree. Will update as -

```PowerShell
& docker run --rm `
    -v /var/run/docker.sock:/var/run/docker.sock `
    -v "${repoRoot}:/repo" `
    -w /repo `
    $ImageBuilderImageName 
    generateTagsReadme "https://github.com/dotnet/dotnet-docker-nightly/blob/${Branch}"
```